styleguide/derek/examples/homepage.ejs

```diff
@@ -2,182 +2,190 @@
-              <hr>
-            </div>
-            <div class="col-md-12">
-              <p class="center">3,000+ hosting engineers across Linux, Windows and VMware</p>
-            </div>
-          </div>
-          <div class="bottom">
-              <a href="/dedicated-servers">Learn More</a>
-          </div>
-        </a>
-      </div>
-      <div class="col-md-3 col-xs-6 product">
-        <a href="/managed-aws">
-          <div class="row">
-            <div class="col-sm-8 col-xs-10 center-block">
-              <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/home/aws.svg">
-            </div>
-          </div>
-          <div class="row">
-            <div class="col-md-12">
-              <h3>Amazon Web Services</h3>
-            </div>
-            <div class="col-xs-3 center-block">
-              <hr>
-            </div>
-            <div class="col-md-12">
-              <p class="center">280+ AWS certified professionals, supporting all 10 AWS public regions</p>
-            </div>
-          </div>
-          <div class="bottom">
-              <a href="/managed-aws">Learn More</a>
-          </div>
-        </a>
-      </div>
-      <div class="col-md-3 col-xs-6 product">
-        <a href="/microsoft">
-          <div class="row">
-            <div class="col-sm-8 col-xs-10 center-block">
-              <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/home/Microsoft2.svg">
-            </div>
-          </div>
-          <div class="row">
-            <div class="col-md-12">
-              <h3>Microsoft Cloud / Azure</h3>
-            </div>
-            <div class="col-xs-3 center-block">
-              <hr>
-            </div>
-            <div class="col-md-12">
-              <p class="center">5-time Microsoft Hosting Partner of the Year — only multi-year winner</p>
-            </div>
-          </div>
-          <div class="bottom">
-              <a href="/microsoft">Learn More</a>
-          </div>
-        </a>
-      </div>
-      <div class="col-md-3 col-xs-6 product">
-        <a href="/openstack">
-          <div class="row">
-            <div class="col-sm-4 col-xs-6 center-block">
-              <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/home/openstack.svg" style="padding-top:9px;">
-            </div>
-          </div>
-          <div class="row">
-            <div class="col-md-12">
-              <h3>Openstack Cloud</h3>
-            </div>
-            <div class="col-xs-3 center-block">
-              <hr>
-            </div>
-            <div class="col-md-12">
-              <p class="center">Co-founder with NASA; the most experience in the world by 100-times</p>
+  </div>
+  <div class="row half-padding-full">
+    <div class="col-md-11 center-block">
+    <a href="[rs_link:managed-security]" class="solution solutions-hoverArea">
```

Comment:
Well I guess we can, but I thought the point of zoolander was to de-drupalize things.   Also its kind of weird when I click on a link in zoolander and get this: 

![image](https://cloud.githubusercontent.com/assets/5263371/16826119/3c4a5cc4-4940-11e6-9ad9-6da9491845be.png)

I kinda like just using `javascript:void(0)` for all links, unless they actually go somewhere on zoolander.  But that may be just me.